### PR TITLE
issue 10558: optimize performance for triple protocol(3.2)

### PR DESCRIPTION
**Pull request by icodening**
## What is the purpose of the change
优化triple协议的性能

## Brief changelog
修改各种Command以支持HTTP2多路复用

PS：测试数据参见 #10558

### CODE REVIEW

**Comment by chickenlj** [comment]
The test failures need to be fixed first.

**Comment by icodening** [comment]
关闭该pr并重新提交至3.1

**Comment by icodening** [comment]
需要解决冲突

**Comment by EarthChen** [comment]
fix ut & it

**Comment by icodening** [comment]
单测与code style已修复，目前差sample没跑通，待修复

File: dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/transport/WriteQueue.java [hunk 1]
```diff
@@ -31,17 +31,31 @@
 public class WriteQueue {
 
     static final int DEQUE_CHUNK_SIZE = 128;
-    private final Channel channel;
+    private volatile Channel channel;
```
Comment (guohao) [inline dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/transport/WriteQueue.java:34]: This `channel` cloud be deleted cuz parent channel can be got from `cmd`'s stream channel

**Review by guohao — CHANGES_REQUESTED** [review]

**Review by guohao — APPROVED** [review]
LGTM

**Review by EarthChen — APPROVED** [review]
LGTM

**Review by AlbumenJ — APPROVED** [review]

**Pull request merged**